bin/build-release.sh

```diff
@@ -67,24 +71,37 @@ function package {(
 
   local tarName="storm-mesos-${RELEASE}.tgz"
   cd _release
-  tar czf ${tarName} apache-storm-${RELEASE}
+  # When supervisor starts up it looks for storm-mesos not apache-storm.
+  mv apache-storm-${RELEASE} storm-mesos-${RELEASE}
```

Comment:
Yes it will. I had to have it in there to make everything work....I figured
I would have to rebase anyway.

On Mon, Jun 1, 2015 at 12:58 PM, Timothy Chen <notifications@github.com>
wrote:

> In bin/build-release.sh
> <https://github.com/mesos/storm/pull/49#discussion_r31445688>:
>
> > @@ -67,24 +71,37 @@ function package {(
> >
> >    local tarName="storm-mesos-${RELEASE}.tgz"
> >    cd _release
> > -  tar czf ${tarName} apache-storm-${RELEASE}
> > +  # When supervisor starts up it looks for storm-mesos not apache-storm.
> > +  mv apache-storm-${RELEASE} storm-mesos-${RELEASE}
>
> Doesn't this conflict with your other PR? That's already merged, so you
> need to rebase
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/storm/pull/49/files#r31445688>.
>